src/main/frontend/multi-pipeline-graph-view/multi-pipeline-graph/main/support/startPollingRunsStatus.ts

```diff
@@ -0,0 +1,27 @@
+import { RunInfo } from "../MultiPipelineGraphModel";
+
+/**
+ * Starts polling the server to retrieve pipeline status.
+ * Will only stop once the run is finished.
+ */
+export default function startPollingRunsStatus(
```

Comment:
Is this just duplicating the logic on https://github.com/jenkinsci/pipeline-graph-view-plugin/pull/14/files#diff-651dc7b464ea9653bba37e54bc7cde19d9a2f5e82510804a9f27a03a5c5e82f3 ? Can't that be repurposed?

Reply:
Do we need live updating on this view? Wondering If it’ll be a performance killer